### ISSUE TITLE
conflict namespace in .\samples\common.h 

### ISSUE THREAD

**Issue by dany137718** [NVIDIA/TensorRT#1892]
hello. 
I was trying to use OpenCV and tensorRt v8.2EA together and saw some issues.
there was a conflict in namespace cv::DataType and nvinfer1::DataType in common.h File. 
I change common.h file and issue solved. (change every DataType to nvinfer1::DataType)
Thanks :
[common.zip](https://github.com/NVIDIA/TensorRT/files/8377380/common.zip)
)

**Comment by ttyio** [comment]
@dany137718  did you include the `common.h` before `using cv` ? Could you try swap the sequence to see if it also fix the issue, thanks

**Comment by dany137718** [comment]
yeah, that works too tanks. but in my project, I couldn't use namespace cv before common.h include.
I test your solution in a simple project and worked. 
Thanks :)

**Comment by ttyio** [comment]
Thanks @dany137718 for confirm, closing
